src/commands/cli/tarballs/smoke.ts

```diff
@@ -54,14 +58,72 @@ export default class SmokeTest extends SfCommand<void> {
       this.execute(executable, '--version'),
       this.execute(executable, '--help'),
       this.execute(executable, 'plugins --core'),
-      this.execute(executable, 'plugins:install @salesforce/plugin-alias@latest'),
+      this.testInstall(executable, '@salesforce/plugin-alias', 'latest'),
     ]);
+    await this.testJITInstall(executable);
     await this.initializeAllCommands(executable);
   }
 
+  private async testJITInstall(executable: string): Promise<void> {
+    const fileData = await fs.promises.readFile('package.json', 'utf8');
```

Comment:
would it make more sense to use this repo's `package.ts` file since that already parses the pjson and has typings for the oclif prop ?

Reply:
@mshanemc I don't think so. `Package` does a lot of additional work like retrieving the package info from npm, which would be unnecessary here